### PR TITLE
Updated roles for admin-service APIs

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: id-authentication-default.properties [hunk 1]
```diff
@@ -590,7 +590,7 @@ mosip.role.keymanager.postdecryptwithpin=ZONAL_ADMIN,GLOBAL_ADMIN,INDIVIDUAL,ID_
 mosip.role.keymanager.postencryptdt=ZONAL_ADMIN,GLOBAL_ADMIN,INDIVIDUAL,ID_AUTHENTICATION,TEST,REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,REGISTRATION_PROCESSOR,PRE_REGISTRATION_ADMIN,RESIDENT
 mosip.role.keymanager.postdecryptdt=ZONAL_ADMIN,GLOBAL_ADMIN,INDIVIDUAL,ID_AUTHENTICATION,TEST,REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,REGISTRATION_PROCESSOR,PRE_REGISTRATION_ADMIN,RESIDENT
 mosip.role.keymanager.postgeneratemasterkeyobjecttype=ZONAL_ADMIN,GLOBAL_ADMIN,INDIVIDUAL,ID_AUTHENTICATION,TEST,REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,REGISTRATION_PROCESSOR,PRE_REGISTRATION_ADMIN,RESIDENT
-mosip.role.keymanager.getgetcertificate=ZONAL_ADMIN,GLOBAL_ADMIN,INDIVIDUAL,ID_AUTHENTICATION,TEST,REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,REGISTRATION_PROCESSOR,PRE_REGISTRATION_ADMIN,RESIDENT
+mosip.role.keymanager.getgetcertificate=ZONAL_ADMIN,GLOBAL_ADMIN,INDIVIDUAL,ID_AUTHENTICATION,TEST,REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,REGISTRATION_PROCESSOR,PRE_REGISTRATION_ADMIN,RESIDENT,KEY_MAKER
```
Comment (ghost) [inline id-authentication-default.properties]: Why are ZONAL_ADMIN, GLOBAL_ADMIN, and REGISTRATION_ADMIN part of this role list?

What about below APIs ?
mosip.role.keymanager.postgeneratecsr
mosip.role.keymanager.postuploadcertificate
mosip.role.keymanager.postuploadotherdomaincertificate

File: kernel-default.properties [hunk 2]
```diff
@@ -454,69 +454,69 @@ mosip.role.admin.masterdata.getbiometrictypes=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN
...
+mosip.role.admin.masterdata.putblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.putblocklistedwordsdetails=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
 mosip.role.admin.masterdata.deleteblocklistedwordsword=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.getblocklistedwordsall=GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordssearch=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordsfiltervalues=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.getblocklistedwordsall=GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordssearch=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordsfiltervalues=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
 mosip.role.admin.masterdata.patchblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,REGISTRATION_ADMIN
```
Comment (ghost) [inline kernel-default.properties]: Is blocklistedWord patch not invoked from UI?

File: kernel-default.properties [hunk 3]
```diff
@@ -548,8 +548,8 @@ mosip.role.admin.masterdata.putlocations=ZONAL_ADMIN,GLOBAL_ADMIN
 mosip.role.admin.masterdata.patchlocations=ZONAL_ADMIN,GLOBAL_ADMIN
 mosip.role.admin.masterdata.deletelocationslocationcode=ZONAL_ADMIN,GLOBAL_ADMIN
 mosip.role.admin.masterdata.getlocationsall=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN,GLOBAL_ADMIN
-mosip.role.admin.masterdata.postlocationssearch=ZONAL_ADMIN,GLOBAL_ADMIN
-mosip.role.admin.masterdata.postlocationsfiltervalues=ZONAL_ADMIN,GLOBAL_ADMIN
+mosip.role.admin.masterdata.postlocationssearch=ZONAL_ADMIN,GLOBAL_ADMIN,REGISTRATION_ADMIN,MASTERDATA_ADMIN
```
Comment (ase-101) [inline kernel-default.properties:551]: why is REGISTRATION_ADMIN added to locations search and filterValues?

File: id-authentication-default.properties (hunk 1 above)
Comment (balaji-alluru) [reply id-authentication-default.properties]: Updated according to the changes suggested.

File: kernel-default.properties (hunk 2 above)
Comment (balaji-alluru) [reply kernel-default.properties]: added MASTERDATA_ADMIN role to blocklistedwords patch

**Review by ghost — APPROVED** [review]

**Pull request merged**